server/Cargo.toml

```diff
@@ -7,11 +7,17 @@ edition = "2021"
 
 [dependencies]
 dotenv = "0.15.0"
-axum = "0.2.8"
+axum = {git="https://github.com/tokio-rs/axum", branch="main"}
```

Comment:
Which features do we get from main, just curious

Reply:
there are some pretty major bugs with middleware on not-main. Basically early-returning middleware do not work in any older version of axum.

Reply:
also fixes compile times of the generic stuff, even without boxed routes

Reply:
axum 0.3 is now available, can we use that instead of pulling from their git